src/Azure/README.md

```diff
@@ -0,0 +1,37 @@
+# ProjectName
+
+These projects enable diagnostics integration for apps deployed in Azure App Service. For additional detail see the [App Service docs](https://learn.microsoft.com/aspnet/core/host-and-deploy/azure-apps/).
+
+## Description
+
+There are two main projects in this area:
+- Microsoft.AspNetCore.AzureAppServicesIntegration
+- Microsoft.AspNetCore.AzureAppServices.HostingStartup
+
+AzureAppServicesIntegration provides the IWebHostBuilder.UseAzureAppServices() API for manually enabling Microsoft.Extensions.Logging to output to App Service's diagnostics infrastructure.
+
+AzureAppServices.HostingStartup Uses an automatic loading mechanism so that the same logging infrastructure can be enabled by only adding this package reference and an environment variable, no code changes are required.
+
+## Development Setup
+
+### Build
+
+To build this specific project from source, follow the instructions [on building the project](../../docs/BuildFromSource.md#step-3-build-the-repo).
```

Comment:
Does this directory require any additional dependencies that would be worthwhile to document here?

Reply:
Also:

```suggestion
To build this specific project from source, follow the instructions [on building the project](../../docs/BuildFromSource.md).
```

I realize we'll need to fix this for the other READMEs as well since I made the updates to the doc. I'll do that shortly...

Reply:
> Does this directory require any additional dependencies that would be worthwhile to document here?

No, it's pretty simple.